### PR TITLE
Add GitHub actions

### PR DESCRIPTION
**Pull request by ball-hayden**
Adds GitHub actions to run tests and prevent issues such as #60.

GitHub actions is free for public repositories.

### CODE REVIEW

**Comment by ball-hayden** [comment]
Bump @cb-yateshmathuria, @cb-goutham (as recent contributors)

**Comment by ball-hayden** [comment]
Bump @cb-yateshmathuria, @cb-prajaktachavan

^ see my comment above. Please let me know if you'd like this PR or not.

**Review by cb-prajaktachavan — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      matrix:
+        ruby-version: ['2.6', '2.7', '3.0']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```
[the remark this answers is not in the document]
Comment (ball-hayden) [reply .github/workflows/main.yml:20]: The issue was that the lockfile had the incorrect version number for `chargebee`.
This is now resolved.

[the remark this answers is not in the document]
Comment (cb-prajaktachavan) [reply .github/workflows/main.yml:20]: Hi @ball-hayden, Thanks for clarifying. Can we remove Gemfile.lock file? Other 2 files looks good to us. Let us know the reason behind committing this lock file.

[the remark this answers is not in the document]
Comment (ball-hayden) [reply .github/workflows/main.yml:20]: Hi @cb-prajaktachavan.

Please see the section here on why `Gemfile.lock` should be committed:
https://bundler.io/guides/faq.html#using-gemfiles-inside-gems

```
Q: Should I commit my `Gemfile.lock` when writing a gem?

A: Yes, you should commit it. The presence of a `Gemfile.lock` in a gem's repository ensures that a fresh checkout of the repository uses the exact same set of dependencies every time. We believe this makes repositories more friendly towards new and existing contributors. Ideally, anyone should be able to clone the repo, run `bundle install`, and have passing tests. If you don't check in your `Gemfile.lock`, new contributors can get different versions of your dependencies, and run into failing tests that they don't know how to fix.
```

**Pull request merged**